virt/virt-2-4-release-notes.adoc

```diff
@@ -4,6 +4,246 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-release-notes
 toc::[]
 
-Reminder: copy the 2.4 release notes into this branch/file to use as a reference for the 2.5 release notes.
+== About Red Hat {VirtProductName}
```

Comment:
Which reminds me, we should add a dummy 2.5 file in the master branch

Reply:
PR to add the dummy 2.5 file: https://github.com/openshift/openshift-docs/pull/26245